lib/Home.dart

```diff
@@ -17,7 +18,7 @@ class Greeting extends StatelessWidget {
             height: 46,
             child: userInfoProvider.hasInfo()
                 ? Text('Hi ${userInfoProvider.info.firstName}!',
-                    style: Theme.of(context).textTheme.headline5)
+                    style: Theme.of(context).textTheme.headline4)
```

Comment:
Why does this not include the CarriageTheme?

Reply:
Discussed offline: remnant of a system I tried before CarriageTheme; Greeting is no longer used and will be removed later